<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>1169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryVoltage">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="velocityLimitX">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>160</y>
      <width>30</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_vel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="odriveConnect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>odriveConnect</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPosX">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>490</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="closedLoopAxis1CheckBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed Loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Up">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>up.png</normaloff>up.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Right">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>right.png</normaloff>right.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Left">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>left.png</normaloff>left.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>down.png</normaloff>down.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Home">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>home.png</normaloff>home.png</iconset>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPos_Up">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPos_Left">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryChargeLevel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SOC</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryVoltage_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Up_Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryVoltage_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>600</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Left_Right</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepInmm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>660</y>
      <width>63</width>
      <height>44</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="batteryVoltage_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step In mm</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_big">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>630</y>
      <width>81</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(181, 181, 181)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 255, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 255, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 255, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 255, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 255, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_BufferEmptied">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>790</y>
      <width>231</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Test To Empty Buffer</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>960</y>
      <width>192</width>
      <height>46</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Auto_Manual">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>133</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <zorder>frame_big</zorder>
   <zorder>progressBar</zorder>
   <zorder>pushButton</zorder>
   <zorder>batteryVoltage</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>velocityLimitX</zorder>
   <zorder>lcd_vel</zorder>
   <zorder>odriveConnect</zorder>
   <zorder>lcdPosX</zorder>
   <zorder>closedLoopAxis1CheckBox</zorder>
   <zorder>pushButton_Up</zorder>
   <zorder>pushButton_Right</zorder>
   <zorder>pushButton_Left</zorder>
   <zorder>pushButton_Down</zorder>
   <zorder>pushButton_Home</zorder>
   <zorder>lcdPos_Up</zorder>
   <zorder>lcdPos_Left</zorder>
   <zorder>batteryChargeLevel</zorder>
   <zorder>label</zorder>
   <zorder>batteryVoltage_2</zorder>
   <zorder>batteryVoltage_3</zorder>
   <zorder>line</zorder>
   <zorder>stepInmm</zorder>
   <zorder>batteryVoltage_4</zorder>
   <zorder>pushButton_BufferEmptied</zorder>
   <zorder>buttonBox</zorder>
   <zorder>pushButton_Auto_Manual</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
